src/main/java/study/step_3/RacingCar.java

```diff
@@ -0,0 +1,26 @@
+package study.step_3;
+
+import study.step_3.view.OutputView;
+
+public class RacingCar {
+
+    private final static int condition = 4;
```

Comment:
```suggestion
    private static final int CONDITION = 4;
```
상수에 대한 컨벤션입니다
(자세한 사항은 아래를 참고해주세요)
> 자바 코드 컨벤션을 지키면서 프로그래밍한다.
참고문서: https://google.github.io/styleguide/javaguide.html 또는 https://myeonguni.tistory.com/1596